src/systems/filecoin_markets/deal/_index.md

```diff
@@ -65,3 +65,34 @@ On every PoSt Submission, the following steps happen.
 # Deal Flow
 
 {{< diagram src="diagrams/deal-flow.mmd.svg" title="Deal Flow Sequence Diagram" >}}
+
+# Deal States
+
+All on-chain economic activities in Filecoin start with the deal. This section aims to explain different states of a deal and their relationship with other concepts in the protocol such as Power, Payment, and Collaterals.
+
+A deal has the following states:
+
+- `Unpublished`: the deal has yet to be posted on chain.
+- `Published`: the deal has been published and accepted by the chain but is not yet active as the sector containing the deal has not been proven.
+- `Active`: the deal has been proven and not yet expired.
+- `Deleted`: the deal or the sector containing the deal has expired or the sector containing the deal has been terminated because of faults.
```

Comment:
can a sector expire without its deals expiring? (if not, remove "or sector")

Reply:
No, a sector expires when all its deals expire. Ok with removing or sector but thought we can make it more explicit.

Reply:
makes it confusing imo